app/build.gradle

```diff
@@ -113,13 +105,17 @@ android {
             matchingFallbacks = ["release"]
         }
 
-        release {
+        internal {
```

Comment:
internal -> releaseFirebase?
verifier only for firebase

Reply:
I asked in discord about that